umm-spec-lib/resources/json-schemas/subscription/umm/v1.0/umm-sub-json-schema.json

```diff
@@ -26,18 +26,20 @@
       "description": "The collection concept id of the granules subscribed.",
       "type": "string",
       "minLength": 1,
-      "maxLength": 255
+      "maxLength": 255 
```

Comment:
Check your editors whitespace cleanup setting. Looks like it's adding spaces

Reply:
I tried to figure this out before - will look more into it.